gdk/wizard/commands/prompt.py

```diff
@@ -0,0 +1,270 @@
+from gdk.wizard.commands.data import Wizard_data
+from gdk.wizard.commons.fields import Fields
+from gdk.wizard.commons.checkers import Wizard_checker
+from gdk.wizard.commons.model import Wizard_model
+import argparse
+from PyInquirer import prompt
+
+
+class Wizard:
+    """
+    A class used to represent the GDK Startup Wizard
+    """
+
+    def __init__(self) -> None:
+        """
+        Initialize the Wizard object
+
+        Attributes
+        ----------
+            data: Wizard_data object
+            getter: Wizard_getter object
+            setter: Wizard_setter object
+            checker: Wizard_checker object
+            parser: argparse object
+        """
+
+        self.data = Wizard_data()
+        self.model = Wizard_model(self.data)
+        self.checker = Wizard_checker(self.data)
+        self.parser = argparse.ArgumentParser()
+
+    def prompter(self, field, value, required, max_attempts=3):
+        """
+        Prompts the user for a value of a given field key
+
+        Parameters
+        ----------
+            field (enum): a field key of the gdk_config file to be changed
+            value (string): the current value corresponding the field key to be changed
+            required (boolean): if the field is a required field of the gdk config file
+            max_attempts (int): the maximum number of attempts to get a valid response from the user
+
+        Returns
+        -------
+            string: the value for the field key "field"
+        """
+        require = "required " if required else "OPTIONAL "
```

Comment:
Any particular reason why required is in small case and optional is in upper case?

Reply:
My apologize, I should have been more consistent. I will change them both to upper case for readability and consistency.